mlflow/pytorch/__init__.py

```diff
@@ -591,6 +680,33 @@ def _load_model(path, **kwargs):
             return torch.jit.load(model_path)
 
 
+@experimental
+def load_state_dict(model_uri):
+    """
+    Load a state dict from a local file or a run.
+
+    :param model_uri: The location, in URI format, of the MLflow model, for example:
+
+                    - ``/Users/me/path/to/local/model``
+                    - ``relative/path/to/local/model``
+                    - ``s3://my_bucket/path/to/model``
+                    - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+                    - ``models:/<model_name>/<model_version>``
+                    - ``models:/<model_name>/<stage>``
+
+                    For more information about supported URI schemes, see
+                    `Referencing Artifacts <https://www.mlflow.org/docs/latest/concepts.html#
+                    artifact-locations>`_.
+    :return: A pytorch model state dict of type OrderedDictionary
+    """
+    import torch
+
+    model_path = _get_model_artifact_path(model_uri)
```

Comment:
Can we just use `_download_artifact_from_uri` here instead of introducing `_get_model_artifact_path`?

```python
local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
```

Reply:
May be in future when we add MLModel file for state dict implementation, we may once again need `_get_model_artifact_path`. I just followed the same convention as in load_model.

Given the fact that we don't have MLModel file at present, we can replace it with`_download_artifact_from_uri`.

Reply:
I have replaced and uploaded a newer verision